integration/dbfs/test_integration.py

```diff
@@ -0,0 +1,124 @@
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint:disable=redefined-outer-name
+import os
+
+import pytest
+
+from databricks_cli.dbfs import cli
+from databricks_cli.dbfs.dbfs_path import DbfsPath
+from tests.utils import invoke_cli_runner
+
+DBFS_TEST_PATH = 'dbfs:/databricks-cli-test'
+LOCAL_TEMP_FILE = 'temp-file.txt'
+LOCAL_TEMP_DIR = 'temp-dir'
+LOCAL_TEST_FILE = 'test-file.txt'
+LOCAL_TEST_DIR = 'test-dir'
+LOCAL_TEST_FILE_IN_DIR = os.path.join(LOCAL_TEST_DIR, LOCAL_TEST_FILE)
+TEST_FILE_CONTENTS = 'Hi I am a test file.\n'
+
+
+@pytest.fixture()
+def dbfs_dir():
```

Comment:
Do we consider the case multiple users running this at the same time? It is safer to create a random directory for each call. Note that the method doesn't really guarantee to clean the folder.